api/src/main/java/science/aist/seshat/FactoryLoader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 the original author or authors.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package science.aist.seshat;
+
+import lombok.SneakyThrows;
+
+import java.util.ServiceLoader;
+
+/**
+ * <p>Factory Pattern implementation for Logger</p>
+ *
+ * @author Andreas Pointner
+ * @since 1.0
```

Comment:
since does not match with the version it was added

Reply:
Change to 1.1